sample/app.yaml

```diff
@@ -43,7 +43,7 @@ spec:
     spec:
       containers:
         - name: traces
-          image: 'ghcr.io/open-telemetry/opentelemetry-collector-contrib/telemetrygen:v0.121.0'
+          image: 'ghcr.io/open-telemetry/opentelemetry-collector-contrib/telemetrygen:v0.122.0'
```

Comment:
Should we keep the `telemetrygen` and the otel image as the same version? Not sure why this was updated but not the otel image

Reply:
By OTel image do you mean the collector image? 
The collector image was updated to v0.122.1. 

v0.122.1 was not available for telemetrygen.

Reply:
Updated the makefile rules in https://github.com/GoogleCloudPlatform/otlp-k8s-ingest/pull/54 to accurately update the version strings in the repo.

Reply:
The updates on app.yaml are currently not automated, and so this file needs to be updated manually. But #54 updates the makefile rule so that GBOC version updates can be automated.